### PR TITLE
Add remote execution jobs to community-templates

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: jobs/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Job Templates
+
+This directory contains job templates for the
+[foreman_remote_execution](https://github.com/theforeman/foreman_remote_execution/)
+plugin.
+
+## Contributing
+
+If you think a job is useful to include here, feel free to open an PR.  Newer
```
Comment (ekohl) [inline jobs/README.md]: Can we name a specific version that started to support this?

Comment (stbenjam) [inline jobs/README.md]: Sure, I'm just not 100% sure of the version number yet.

**Comment by mmoll** [comment]
As I'm not really following r_ex development at the moment: Is theforeman/foreman_templates#23 required for this to be really useful?

**Comment by stbenjam** [comment]
> As I'm not really following r_ex development at the moment: Is theforeman/foreman_templates#23 required for this to be really useful?

I don't think so, we definitely want that in foreman_templates, but this was more to encourage contributions.

More than a few people have asked where they could contribute additional templates/submit PR's, and it's a lot easier if it's all in the community-templates repo. 

 `foreman_remote_execution` requires redmine tickets, and just overall a higher bar to entry.

**Comment by stbenjam** [comment]
Also if you are all OK with this change, it'd be good if some of the REX people had commit here. I don't at least, not sure about @ares.  But I suppose we can deal with that when we actually get a PR from someone.

**Comment by mmoll** [comment]
@ares does have push :open_hands: permissions here.
I'd say this is OK to merge once r_ex nightlies can export jobs and I would be highly fond of also having the import functionality in foreman_templates.

File: jobs/README.md (hunk 1 above)
Comment (ekohl) [inline jobs/README.md]: If it's not released yet then this is fine for now. We'll fix it later.

**Comment by ekohl** [comment]
I'd be fine with giving @stbenjam merge permissions as well.

**Comment by mmoll** [comment]
Given the Puppet work, @stbenjam is doing over at Katello, I'd also have no objections adding him the the installer team here. :+1:

**Comment by stbenjam** [comment]
Updated, 0.3.2 will be released soon (package PR's are open) and includes export/import.

If this is good to go, I'd appreciate a merge before the demo this morning. It's not support important, though.

**Comment by mmoll** [comment]
Sure thing, merged, thanks @stbenjam! :icecream:

**Pull request merged**